### DISCUSSION TITLE
Is it possible to obtain Initializer info from IPropertySymbol or IFieldSymbol?

### DISCUSSION THREAD

**Discussion by Stoffelche** [dotnet/roslyn#66461] [Q&A] [answered]
Is this info available on the symbols or does one have to resort to the SyntaxNode to obtain Initializer information?

**Comment by CyrusNajmabadi** [comment, marked as answer]
Same answer as https://github.com/dotnet/roslyn/discussions/66460

For source, you go back to the node. For  metadata there is no way to tell, as there is no info stored there for this information.

**Reply by Stoffelche** [reply]
Thank you
